include/sentry.h

```diff
@@ -728,6 +728,18 @@ SENTRY_API void sentry_options_set_ca_certs(
 SENTRY_API const char *sentry_options_get_ca_certs(
     const sentry_options_t *opts);
 
+/**
+ * Configures the name of background worker thread
+ */
+SENTRY_API void sentry_options_set_bgworker_name(
```

Comment:
```suggestion
 * Configures the name of the http transport thread.
 */
SENTRY_API void sentry_options_set_transport_thread_name(
```
… and the same below. Just in case we introduce other threads in the future.